client/src/components/course-view/EditAssessmentModelDialog.tsx

```diff
@@ -0,0 +1,100 @@
+// SPDX-FileCopyrightText: 2023 The Aalto Grades Developers
+//
+// SPDX-License-Identifier: MIT
+
+import {
+  Button,
+  CircularProgress,
+  Dialog,
+  DialogActions,
+  DialogContent,
+  DialogTitle,
+  TextField,
+} from '@mui/material';
+import {enqueueSnackbar} from 'notistack';
+import {JSX, useEffect, useState} from 'react';
+import {useParams} from 'react-router-dom';
+
+import {useEditAssessmentModel} from '../../hooks/useApi';
+
+type EditAssessmentModelDialogProps = {
+  assessmentModelId: number | null;
+  name: string | null;
+  onClose: () => void;
+  open: boolean;
+};
```

Comment:
Have you decided on any consistent style for writing prop types? As in, in some places they are defined as `type PropsType = {...}`, here it is defined as `type ComponentNameProps = {...}`, and in others it's just defined in the component `const ComponentName = (props: {...}) =>`

Reply:
Until the types are not exported, and only one component exists in a single file PropsType should be good. This case is not always respected though.